atc/db/migration/migrations/1593541563_add_latest_successful_build_id.up.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
+    ALTER TABLE jobs ADD COLUMN latest_successful_build_id integer;
```

Comment:
I'm not a huge fan of nullable fields; there's no way to know for certain why they are null. To express absence-of-value, I think tables work better. So instead of adding a column to `jobs`, it'd be something like this:

```
CREATE TABLE last_successful_build_in_job (
  job_id integer references jobs(id),
  build_id integer references builds(id),
  primary key (job_id, build_id),
  constraint job_uk unique (job_id)
);
```